src/Storages/StorageView.cpp

```diff
@@ -266,7 +267,8 @@ String StorageView::replaceQueryParameterWithValue(const String & column_name, c
         if ((pos = name.find(parameter.first)) != std::string::npos)
         {
             auto parameter_datatype_iterator = parameter_types.find(parameter.first);
-            if (parameter_datatype_iterator != parameter_types.end())
+            size_t parameter_end = pos+parameter.first.size();
```

Comment:
to stick to the style guide
```suggestion
            size_t parameter_end = pos + parameter.first.size();
```

Reply:
I would also suggest to avoid using both `String` and `std::string` in the same file/function

Reply:
Sorry I didnt get this, they are same right ?
https://github.com/ClickHouse/ClickHouse/blob/0d45443cb06abb8c11a3488fd795bad4e052b54e/base/base/types.h#L40

Reply:
Correct, but imho it is better not to mix names. It is a little bit confusing (also just about style)

Reply:
got it. I might be refactoring some parts of parameterized view, will keep this in mind, thanks